scripts/konlet-startup

```diff
@@ -28,6 +28,28 @@ if [[ $? -ne 0 ]]; then
   ACTUAL_KONLET_VERSION="${KONLET_VERSION_TAG}"
 fi
 
+sudo sh -c 'echo "[Unit]
+Description=A daemon to collect Docker events and send to to journal logs.
```

Comment:
I'd very much rather see this just included in COS image, but disabled by default.

Reply:
We prefer to have a more robust solution than a systemd script. i.e. We need the solution to handle Docker daemon restart properly. For context: b/79548044#comment9